README.md

```diff
@@ -66,46 +62,44 @@ E.g. Do you need to upgrade to Symfony 4.0?
     vendor/bin/rector process src --config your-own-config.yml
     ```
 
-2. Do you want to see preview of changes first? 
+2. Do you want to see the preview of changes first?
 
-    Use `--dry-run` option:
-    
-    ```
+    Use the `--dry-run` option:
+
+    ```bash
     vendor/bin/rector process src --level symfony33 --dry-run
     ```
 
-
 ### B. Custom Sets
 
-1. Create `rector.yml` with desired Rectors
+1. Create `rector.yml` with desired Rectors:
 
     ```yml
     rectors:
         - Rector\Rector\Contrib\Nette\Application\InjectPropertyRector
     ```
 
-2. Try Rector on your `/src` directory
+2. Try Rector on your `/src` directory:
 
     ```bash
     vendor/bin/rector process src --dry-run
     ```
 
-3. Apply Changes if you like them
+3. Apply the changes if you like them:
 
-    ```
+    ```bash
     vendor/bin/rector process src
     ```
 
-
-### Simple setup with Dynamic Rectors
+## Simple setup with Dynamic Rectors
 
 You don't have to always write PHP code. Many projects change only classes or method names, so it would be too much work for a simple task.
 
 Instead you can use prepared **Dynamic Rectors** directly in `*.yml` config:
 
 You can:
 
-- **replace class name**
+- **Replace a class name**:
```

Comment:
I find these double colons rather stopping the eyes from reading then helping. Could you remove them?